convert2rhel/cert.py

```diff
@@ -58,3 +60,33 @@ def install(self):
             loggerinst.critical("OSError({0}): {1}".format(err.errno, err.strerror))
 
         loggerinst.debug("Certificate copied to %s." % self._system_cert_dir)
+
+    def remove(self):
+        """Removes certificate (.pem), which was copied in install method"""
```

Comment:
```suggestion
        """Remove certificate (.pem) which was copied to system's cert dir"""
```

Reply:
Just FYI, the `Removes` -> `Remove` change comes from https://www.python.org/dev/peps/pep-0257/:
> [The docstring] prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".